src/plugins/api/apitypes.h

```diff
@@ -481,18 +481,18 @@ enum class NoteHeadGroup {
 };
 Q_ENUM_NS(NoteHeadGroup);
 
-enum class NoteType : char {
+enum class NoteType : unsigned char {
```

Comment:
Maybe we should change it to `uint8_t`. It's shorter 😉
We can also simply remove it (so that we get `int`). In fact, `mu::engraving::NoteType` is `int` too, so I doubt that we gain much by using a different type for the api version.

Reply:
OK, let's remove it then